packages/core/src/js/sdk.tsx

```diff
@@ -67,8 +67,15 @@ export function init(passedOptions: ReactNativeOptions): void {
       return undefined;
     }
     try {
-      const url = new URL(dsn);
-      return `${url.protocol}//${url.host}`;
+      const regex = /^(https?):\/\/(?:[^@]+@)?([^/]+)(?:\/.*)?$/;
+      const matches = dsn.match(regex);
+
+      if (matches) {
+        const [, protocol, host] = matches;
```

Comment:
I've missed that in the original review as well, but let's use the `makeDsn` function from `sentry/core`. It already uses regex and is used in the client impl, so it 100% works with RN.

https://github.com/getsentry/sentry-javascript/blob/6f56d063e479f90ce07a5ddc0636dc0bc3b3bb2e/packages/core/src/utils-hoist/dsn.ts#L122

Reply:
Good idea @krystofwoldrich 👍 
Updated with a1ecb49a4165bed7106f84327303529d844b57f1